grails-app/views/dashboard/_catalogsSummary.gsp

```diff
@@ -0,0 +1,108 @@
+            <g:if test="${params.editCatalogs}">
+                <g:isUserAdmin>
+                    <table>
+                        <thead>
+                            <tr>
+                                <th><warehouse:message code="catalogs.name.label" default="Formulary"/></th>
+                                <th><warehouse:message code="tag.count.label" default="Count"/></th>
+                                <th><warehouse:message code="tag.active.label" default="Active"/></th>
+                                <th><warehouse:message code="default.actions.label"/></th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <g:each in="${catalogs }" var="catalog" status="i">
+                                <tr class="${i%2?'odd':'even'}">
+                                    <td>
+                                        ${catalog.name?:"Empty catalog"}
+                                    </td>
+                                    <td>
+                                        ${catalog.productCatalogItems.size()}
+                                    </td>
+                                    <td>
+                                        ${catalog.active}
+                                    </td>
+                                    <td>
+                                        <g:link controller="dashboard" action="hideCatalog" id="${catalog}" params="[editCatalogs:true]">
+                                            <img src="${createLinkTo(dir:'images/icons/silk',file:'bullet_cross.png')}"/></g:link>
+                                    </td>
+                                </tr>
+                            </g:each>
+                        </tbody>
+                    </table>
+                </g:isUserAdmin>
+            </g:if>
+            <g:else>
+                <g:if test="${catalogs}">
+                    <div id="tagcloud">
+                        <g:each in="${catalogs }" var="catalog">
+                            <g:if test="${catalog?.productCatalogItems?.size() > 1}">
+                                <g:link controller="inventory" action="browse" params="['catalogs':catalog.id]" rel="${catalog?.productCatalogItems?.size() }">
+                                    ${catalog.name?:"Empty catalog" } (${catalog?.productCatalogItems?.size() })</g:link>
+                            </g:if>
+                        </g:each>
+                    </div>
+                </g:if>
+                <g:else>
+                    <div style="margin:10px;" class="center">
+                        <span class="fade"><warehouse:message code="catalogs.noCatalogs.label"/></span>
+                    </div>
+                </g:else>
+            </g:else>
+        </div>
+		<div class="clear"></div>
+	</div>
+</div>
+<script src="${createLinkTo(dir:'js/jquery.tagcloud', file:'jquery.tagcloud.js')}" type="text/javascript" ></script>
```

Comment:
Move the js file to the main dashboard/index.gsp page since it'll be included twice (once for tags, once for catalogs).